src/client/swagger/unity.yml

```diff
@@ -23,6 +23,35 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  '/me/{id}':
```

Comment:
Could the endpoint be `/me` with no ID? I think we would already know the login user ID from the cookie on the request.

Reply:
Definitely, if that's the case then for sure